### PR TITLE
tools: fix typos in the code

### PR DESCRIPTION
**Pull request by c8ef**
This patch corrects a minor typo without any functional changes.

### CODE REVIEW

**Comment by nyh** [comment]
@c8ef thanks for the patch. I know this requirement is sort of silly for such small typo fix, but as CONTRIBUTING.md explains, before we merge this patch we will need:
1. You will need to sign https://www.scylladb.com/open-source-nosql-database/contributor-agreement/ saying that you officially agree to contribute code to this project.
2. You will need to use a real name to sign this (and also in the pull request).

File: tools/scylla-sstable.cc [hunk 1]
```diff
@@ -671,7 +671,7 @@ class writetime_histogram_collecting_consumer : public sstable_consumer {
                 }
             });
         } else {
-            throw std::runtime_error(fmt::format("Cannot collect timestamp of cell (column {} of uknown type {})", cdef.name_as_text(), cdef.type->name()));
```
Comment (avikivity) [inline tools/scylla-sstable.cc:674]: > fix typo;
> Signed-off-by: Yuao Ma <c8ef@outlook.com>

This repository expects better changelog messages, use `git log` to get a sense of how they are formatted.

Comment (avikivity) [reply tools/scylla-sstable.cc:674]: Also, please set the patch author to your name.

**Review by denesb — COMMENTED** [review]
>  typos: fix typos in code 

The prefix (the part before the `:`) denotes the subsystem changed.
In the case of this patch, it should be `tools: fix typos in the code`.

**Review by denesb — APPROVED** [review]

**Comment by denesb** [comment]
Kicked of CI.

**Comment by denesb** [comment]
Queued.

**Pull request closed without merging**